test/utils/test_serialization.py

```diff
@@ -10,8 +10,8 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-from os import remove
-from tempfile import mkstemp
+import os
```

Comment:
I try to avoid unnecessary imports in the the entire module by `from os.path import join` (maybe `as join_path`)

Reply:
changed